src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/Dashboard.xaml.cs

```diff
@@ -340,6 +342,22 @@ private void Commit()
                 var notificationService = _model.Session.Workspace.Services.GetService<INotificationService>();
```

Comment:
We're getting into two different ways to report problems to a user now. `INotificationService` will pop up a dialog right?

Reply:
Basically we just need to decide if failure of an action from the user (i.e commit rename) should show a dialog or yellow bar. In most dialog driven cases I think we continue to use dialogs as a means to convey errors. Inline rename is a little different in that the dialog is not a modal dialog, but maybe we should keep the same?

Reply:
As mentioned offline, I prefer the gold bar for an unexpected exception as it doesn't interupt the user for something they can't necessarily do anything about, and might encourage more reports if this is a common failure, which we can maybe provide a better fix for.

I'll leave this for now, but happy to follow up if there is more feedback.